courses/free-courses-en.md

```diff
@@ -740,8 +740,11 @@
 * [Intro to Self-Driving Cars Nanodegree](https://www.udacity.com/course/intro-to-self-driving-cars--nd113) - Cezanne Camacho, Andrew Paster, Anthony Navarro, Tarin Ziyaee, et al. (Udacity)
 * [Introduction to Machine Learning](https://openlearninglibrary.mit.edu/courses/course-v1:MITx+6.036+1T2019/about) - MIT Open Learning Library
 * [Machine Learning](https://www.sololearn.com/Course/machine-learning) (SoloLearn) *(account required)*
+* [Machine Learning](https://youtube.com/playlist?list=PL7T06JEc5PF5Vuz1U7lEEaFPOExDbKVEh) - Nerd's Lesson (YouTube)
```

Comment:
Instructor name is `Pedro Domingos` and channel is `Nerd's Lesson`. How to treat this @eshellman ?

Reply:
just use `Nerd's Lesson`. The rule for youtube is to fallow the example of the resource - if the creators don't want to highlight their personal name, we shouldn't feel the need to over-rule.

Also, we are no longer adding `(YouTube)` on the courses pages - it's almost a default.